Name: Change ReplyEmail Settings
Version: 1.3.1-1
Author: mikepre
Description: This patch is currently not compatible with Replies/Forwards in Black Font. Please remove before you try to apply this patch. Changes the Reply to Font to Calibri, Size 15, Black. Also changes the Reply to remove the "John Smith wrote" text and replace it with the typical "From: ... To: ..." language that Outlook displays.

--- .orig/usr/palm/applications/com.palm.app.email/app/models/Email.js
+++ /usr/palm/applications/com.palm.app.email/app/models/Email.js
@@ -193,9 +193,22 @@
 				this.text += "<br/>";
 			} else {
 				this.text = Email.kReplyForwardSetupElems;
-				this.text += $L("#{displayName} wrote:").interpolate(fromObj) + "<br/><br/>";
-			}
-			this.originalText = original.text;
+				if (fromObj.displayName === fromObj.address) {
+					this.text += $L("<b>From:</b> ") + "#{displayName}".interpolate(fromObj).escapeHTML() + "<br/>";
+				}
+				else {
+					this.text += $L("<b>From:</b> ") + "#{displayName} <#{address}>".interpolate(fromObj).escapeHTML() + "<br/>" 
+				};
+				this.text += $L("<b>Date:</b> ") + Mojo.Format.formatDate(new Date(parseInt(original.timeStamp)), {date:'medium', time:'short'}) + "<br/>";
+				this.text += $L("<b>Subject:</b> #{summary}").interpolate(original) + "<br/>";
+				if (toAddressList.length > 0) {
+					this.text += $L("<b>To:</b> ") + "#{list}".interpolate({list:toAddressList.join("; ")}).escapeHTML() + "<br/>";
+				}
+				if (ccAddressList.length > 0) {
+					this.text += $L("<b>CC:</b> ") + "#{list}".interpolate({list:ccAddressList.join("; ")}).escapeHTML() + "<br/>";
+				}
+				this.text += "<br/>";
+			}			this.originalText = original.text;
 		} else {
 			this.text = "";
 			this.originalText = "";
@@ -488,8 +501,8 @@
 	return request;
 }
 
-Email.kSignaturePlaceholder = "<span style='font-family:Prelude, Verdana, san-serif;'><br><br></span><span id='signature'></span>";
-Email.kReplyForwardSetupElems = "<span style='color:navy; font-family:Prelude, Verdana, san-serif; '>" + Email.kSignaturePlaceholder + "<hr align='left' style='width:75%'/>";
+Email.kSignaturePlaceholder = "<span style='font-family:Prelude, Calibri, san-serif;  font-size:15px'><br><br></span><span id='signature'></span>";
+Email.kReplyForwardSetupElems = "<span style='color:black'>" + Email.kSignaturePlaceholder + "<hr align='left' style='width:75%'/>";
 
 Email.kSubjectPrefixRe = $L("Re: ");
 Email.kSubjectPrefixFw = $L("Fw: ");
--- .orig/usr/palm/applications/com.palm.app.email/app/controllers/compose-assistant.js
+++ /usr/palm/applications/com.palm.app.email/app/controllers/compose-assistant.js
@@ -712,7 +712,8 @@
 	var editor = controller.get('msg_rte');
 	if (email.action == "forward") {
 		controller.get('forwarded_msg_body').update(originalText);
-	} else {
+	} 
+else {
 		bodyText += originalText;
 		editor.setStyle({'min-height':'320px', 'font-size':'18px'});
 	}
@@ -718,7 +719,7 @@
 	}
 	editor.update(bodyText);
 
-	if (email.highlightBody) {
+		if (email.highlightBody) {
 		editor.focus();
 //		var sel = controller.window.getSelection();
 //		sel.selectAllChildren(editor);
